### PR TITLE
Editor Prototyping

### PR DESCRIPTION
**Pull request by jcontonio**
* Adds content editor at http://localhost:3000/dashboard
  * Try writing in markdown for italics, bold, and headings and things like that
* Adds password reset, more/smarter partials, and adding meta tags / images to quiet the 404's

I need help with the RegEx's. You'll see that the bold tags are getting hi-jacked by the italic pattern. It's in EditorComponent.js, line 42

### CODE REVIEW

**Comment by jcontonio** [comment]
image

File: web/src/shared/utilities.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export const debounce = function(delay, fn) {
```
Comment (dheniges) [inline web/src/shared/utilities.js:1]: Don't think this is used anywhere?

**Review by dheniges — DISMISSED** [review]
Looks good except maybe one unused function?

If we switch editors that's cool, but might as well have something in there at the moment

File: web/src/shared/utilities.js (hunk 1 above)
Comment (jcontonio) [reply web/src/shared/utilities.js:1]: It's not used, but we will definitely use it - should leave it in

**Review by jjidt — APPROVED** [review]
👍

**Pull request merged**